tests/core/pyspec/eth2spec/test/phase0/genesis/test_initialization.py

```diff
@@ -1,14 +1,29 @@
-from eth2spec.test.context import PHASE0, spec_test, with_phases, single_phase
+from eth2spec.test.context import (
+    PHASE0, ALTAIR, MINIMAL,
+    is_post_altair,
+    single_phase,
+    spec_test,
+    with_configs,
+    with_phases,
+)
 from eth2spec.test.helpers.deposits import (
     prepare_full_genesis_deposits,
     prepare_random_genesis_deposits,
 )
 
 
-@with_phases(([PHASE0]))
+def get_post_altair_description(spec):
+    return f"Although it's not phase 0, we may use {spec.fork} spec to start testnets."
+
+
+@with_phases(([PHASE0, ALTAIR]))
```

Comment:
Should we even specify the phase list? instead of all by default?

Reply:
I somehow had an impression that `PHASE1` doesn't work for genesis tests. But it turns out `@with_all_phases` work well. Thanks!